apps/emqx/src/persistent_session/emqx_persistent_session.erl

```diff
@@ -291,8 +274,6 @@ resume(ClientInfo = #{clientid := ClientID}, ConnInfo, Session) ->
     %%    when the messages were delivered.
     ?tp(ps_persist_pendings, #{sid => SessionID}),
     Session1 = emqx_session:enqueue(ClientInfo, Pendings2, Session),
-    Session2 = persist(ClientInfo, ConnInfo, Session1),
-    mark_as_delivered(SessionID, Pendings2),
```

Comment:
Q: why don't we need to mark them as delivered anymore?

Reply:
We need, but that's now a part of `emqx_session:enqueue/3` (see `persist_delivers/2` there).

Reply:
True, ignore me.  :speak_no_evil: